pkg/cmd/cli/cmd/login.go

```diff
@@ -79,6 +85,13 @@ func NewCmdLogin(fullName string, f *osclientcmd.Factory, reader io.Reader, out
 				os.Exit(1)
 
 			} else {
+				if os.IsPermission(err) {
```

Comment:
I like having this, but I'd also like a pre-check.  The pre-check doesn't have to be perfect, but even 80% right would be really nice.